datahub-web-react/src/app/lineage/utils/lineageUtils.ts

```diff
@@ -3,10 +3,10 @@ import dayjs from 'dayjs';
 const MILLIS_PER_HOUR = 3600000;
 
 /**
- * Returns the default time-lineage start time which is 14 days - current time, rounded down to the nearest hour.
+ * Returns the default time-lineage start time which is 28 days - current time, rounded down to the nearest hour.
  */
 export const getDefaultLineageStartTime = () => {
-    return Math.floor(dayjs().subtract(14, 'day').valueOf() / MILLIS_PER_HOUR) * MILLIS_PER_HOUR;
+    return Math.floor(dayjs().subtract(28, 'day').valueOf() / MILLIS_PER_HOUR) * MILLIS_PER_HOUR;
```

Comment:
Can we make this a configurable value, instead of hardcoded?

Reply:
We can but it adds backend changes to this PR. I would suggest we break into a separate PR given the additional scope.

Reply:
I agree with this

Reply:
Then I would ask why 28 and why not 31 or 35 (to accommodate monthly jobs)

Reply:
yeah I'd prefer 30 days instead of 4 weeks